.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "assets/css"]
 	path = assets/css
-	url = git@github.com:Glimesh/glimesh.tv-styles.git
+	url = ../../Glimesh/glimesh.tv-styles.git
```

Comment:
Is this right?

Reply:
I would try it, but it worked for me in WSL 2 and in Manjaro.

Reply:
Confirmed by cloning a new repository and trying to spin up. Also found a StackOverflow question confirming my solution: https://stackoverflow.com/questions/40841882/automatically-access-git-submodules-via-ssh-or-https

Reply:
Oh interesting, so it back two directories on GitHub... Not on your local filesystem.